pages/common/kube-capacity.md

```diff
@@ -0,0 +1,16 @@
+# kube-capacity
+
+> This is a simple CLI that provides an overview of the resource requests, limits, and utilization in a Kubernetes cluster.
```

Comment:
Perhaps this will read better?
```suggestion
> A simple CLI that provides an overview of the resource requests, limits, and utilization in a Kubernetes cluster.
```